pybedtools/bedtool.py

```diff
@@ -1969,25 +1994,29 @@ def subtract(self):
 
             >>> c = a.subtract(b, f=0.5)
         """
-        kwargs["b"] = b
-
         if "a" not in kwargs:
             kwargs["a"] = self.fn
 
+        if "b" not in kwargs:
```

Comment:
I might be wrong but I beleive this code was incorrect as b is being referenced without ever being given... maybe this is a result of the @_wraps decorator?

I could not understand it... and I tried to correctly pick up the b from kwargs... please forgive me if this is incorrect